scanner/src/funTest/kotlin/scanners/AbstractScannerTest.kt

```diff
@@ -76,27 +77,25 @@ abstract class AbstractScannerTest : StringSpec() {
 
     init {
         "Scanning a single file succeeds".config(tags = testTags) {
-            val resultsFile = outputDir.resolve("${scanner.scannerName}.${scanner.resultFileExt}")
+            val result = scanner.scanInputPath(inputDir.resolve("LICENSE"), outputDir)
+            val summary = result.scanner?.results?.scanResults?.singleOrNull()?.results?.singleOrNull()?.summary
```

Comment:
Maybe you could shorten this by using `OrtResult.getScanResultsForId(id: Identifier)`. This would also verify the generated identifier, but not that there is only a single result.

Reply:
As this is scanning a path, the identifier is a bit cumbersome to construct. In fact, I just realized we are using the OS as the `type` and the absolute path as the `namespace`... probably something that could be improved. Anyway, I somehow like this long line as it shows how terribly nested our scan result structure is 🙄